### ISSUE TITLE
UI for Implementation of scheduling & e-mailing reports

### ISSUE THREAD

**Issue by Ippezrobert** [openMF/community-app#1867]

**Comment by edcable** [comment]
@Ippezrobert  Could you provide some more details or even draft a wireframe of what the UI should support in terms of scheduling/emailing reports.

**Comment by Ippezrobert** [comment]
**UI Mockup for Email Report Scheduler**

**Email Report Scheduler Landing Screen**
![reportmailingjobs](https://cloud.githubusercontent.com/assets/13125234/24138669/37340500-0e2a-11e7-8e7f-2983489d1e7d.png)

**Create and Edit Email Report Scheduler**
![create and edit reportmailingjobs](https://cloud.githubusercontent.com/assets/13125234/24138675/3cfd8132-0e2a-11e7-88ec-17670937edd8.png)

**View Email Report Scheduler**
![view reportmailingjobs](https://cloud.githubusercontent.com/assets/13125234/24138680/41f69502-0e2a-11e7-92d0-9e56cab0d63d.png)

**Comment by julianmojico** [comment]
I'm wondering what is the status of this.

**Comment by awasum** [comment]
@cepix  Go ahead and work on this if you want. I dont think anyone is working on it.
 @santoshconflux  ?

**Comment by santoshconflux** [comment]
@awasum , @cepix , Yes, no one is working on it. It will be very good feature once it is implemented. Currently, we are configuring it in database level.

**Comment by awasum** [comment]
See backend issue on Fineract: https://issues.apache.org/jira/browse/FINERACT-556 and PR already merged: https://github.com/apache/fineract/pull/428
I dont know if the backend side is already complete.
